src/renderer/components/watch-video-info/watch-video-info.sass

```diff
@@ -0,0 +1,72 @@
+.watchVideoInfo
+  display: grid
+  grid-template-columns: 2fr 1fr
+  padding: 16px
+
+  @media screen and (max-width: 680px)
+    grid-template-columns: auto
+
+.videoTitle
+  font-size: 22px
+  margin: 0 0 20px
+
+.channelInformation
+  .profileRow
+    display: flex
+
+  .channelThumbnail
+    border-radius: 50%
+    margin-right: 8px
+
+  .channelName
+    margin-left: 6px
+
+  .subscribeButton
+    padding: 5px
+    font-size: 14px
+
+.datePublished
+  margin-bottom: 24px
+
+.viewCount
+  margin: 4px 0px 0px
+  text-align: right
+
+  @media screen and (max-width: 680px)
+    margin-top: 16px
+    text-align: left
+
+.likeSection
+  margin-top: 16px
+  font-size: 14px
+  display: flex
+  flex-direction: column
+  margin-left: auto
+  text-align: right
+  max-width: 210px
+
+  @media screen and (max-width: 680px)
+    margin-left: 0
+
+  .likeBar
+    height: 8px
+    border-radius: 4px
+    margin-bottom: 4px
+
+  .likeCount
```

Comment:
Let's return the style of the like / dislike count back to what it is in `master`.  Font size is 12px and the color is `var(--tertiary-text-color)`

Reply:
Wow, you're right, that looks so much better!